src/Illuminate/Routing/PendingResourceRegistration.php

```diff
@@ -143,6 +143,19 @@ public function middleware($middleware)
     }
 
     /**
+     * Set the resource to be a singular resource instance.
+     *
+     * @param  bool|string|array  $resources
+     * @return \Illuminate\Routing\PendingResourceRegistration
```

Comment:
All of the other `@return` statements for fluent methods in this class use the FQCN. Using `@return $this` on this method would be inconsistent with the rest of the class.

We could change all the rest of the methods to use `@return $this`, but I think that would belong in another PR.